src/test/unit/test_tracer.py

```diff
@@ -439,3 +439,17 @@ def lambda_test_function(event, context):
         aws_secret_access_key=secret_access_key,
         config=ANY,
     )
+
+
+def test_get_stacktrace(context):
+    @lumigo_tracer(token="123")
+    def lambda_test_function(event, context):
+        raise Exception("Inner exception")
+
+    with pytest.raises(Exception):
+        lambda_test_function({}, context)
+
+    function_span = SpansContainer.get_span().function_span
+    assert function_span["error"]["type"] == "Exception"
+    assert "lumigo_tracer/tracer.py" not in function_span["error"]["stacktrace"]
+    assert 'raise Exception("Inner exception")' in function_span["error"]["stacktrace"]
```

Comment:
Because we format the exception, the exception is critical for the client and there is the issue with `\n`, IMO it's worth having a test that checks the whole string. You can generate the "expected" string by running a function that isn't wrapped by the tracer.